### ISSUE TITLE
OSX - 0.24.1 - unable to use latest flash player

### ISSUE THREAD

**Issue by Blatman** [nwjs/nw.js#6072]
OSX 10.11.6  El Capitan

Adobe appear to have changed the layout of the Flash plugin.
I noticed the latest system install placed Flash Player.plugin into the Macintosh HD/Library/Internet Plug-Ins folder whereas the previous system installs were bundled in a PepperFlashPlayer folder.
Naming has also altered from PepperFlashPlayer to Flash Player.
If the nw.js app has no PepperFlash installed it will still use the old system installed PepperFlash (will give an out of date message but can be run) however it will not pick up the latest Flash install. I am also unable to use the latest Flash Player.plugin with the nw.js app so at this stage can only use legacy PepperFlash.
Tried flash in Chrome (pre-59 and 59) and it picks up the latest Flash Player.plugin just fine.

[PepperFlashTest.zip](https://github.com/nwjs/nw.js/files/1193022/PepperFlashTest.zip)

**Comment by Christywl** [comment]
Seem it works fine for me on Mac with nwjs-sdk-v0.24.1. My PepperFlash version is 26.0.0.137. When open http://get.adobe.com/flashplayer/about/, it works fine.

**Comment by Blatman** [comment]
OK, I will be unable to troubleshoot further for a wee while but I will try it on another machine to see what the system flash install looks like. Does your install still use the PepperFlashPlayer folder rather than Flash Player.plugin?

**Comment by Christywl** [comment]
`/Library/Internet Plug-Ins/PepperFlashPlayer/PepperFlashPlayer.plugin` It's installed by default when I install Chrome on Mac.

**Comment by rogerwang** [comment]
close until more information is available.

**Comment by Blatman** [comment]
OK, just back and tried some more tests - (for Info)
Tried testproject on 10.10.5 (Yosemite) and it also failed. The Yosemite machine only had /Library/Internet Plug-Ins/Flash Player.plugin as the system install. I uninstalled this using the Adobe uninstaller and installed latest Chrome. Chrome appears to install flash somewhere (not /Library/Internet Plug-Ins/ and when run and enabled places a copy of the PepperFlash folder in the Chrome Profile. At this point the test project would not work until I took a copy of the PepperFlash folder from the Chrome profile and placed it in the system location as /Library/Internet Plug-Ins/PepperFlashPlayer/. I also had to shift PepperFlashPlayer.plugin and manifest.json up into the PepperFlashPlayer folder. Things worked perfectly then.
I then installed flash from Adobe and it installed Flash Player.plugin into /Library/Internet Plug-Ins/ but NW.js won't work with this. Both my machines had Adobe flash installs and NW.js appears to need the PepperFlashPlayer folder. 
In summary it looks to me as though if fresh Adobe Flash installs may be an issue but since PepperFlash is again available within Chrome then that can be used for NW.js

Cheers.
